pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -44,20 +44,26 @@ import (
 	"istio.io/pkg/log"
 )
 
+// ExternalCaType : External CA Integration type
+type CaExternalType string
+
 type caOptions struct {
 	// Either extCAK8s or extCAGrpc
-	CAType       string
-	CASignerName string
+	ExternalCAType   CaExternalType
+	ExternalCASigner string
 	// domain to use in SPIFFE identity URLs
 	TrustDomain    string
 	Namespace      string
 	Authenticators []authenticate.Authenticator
 }
 
 const (
-	//KubernatesCA : Kubernates acts as the Certificate Authority
-	extCAK8s string = "KUBERNATES"
-	// extCAGrpc string = "GRPC"
+	// ExtCAK8s : Integrate with external CA using k8s CSR API
+	ExtCAK8s CaExternalType = "ISTIOD_RA_KUBERNETES_API"
```

Comment:
"ISTIOD_**C**A_KUBERNETES API"?

Reply:
nevermind, just got the context around Registration Authority :)

Reply:
Yeah, had a lot of back and forth regarding the nomenclature. This is the one we settled on